Makefile

```diff
@@ -49,3 +43,14 @@ clean:
 	rm -rf zookeeper-*/
 	rm -f zookeeper
 	rm -f profile.cov
+
+.PHONY: jute
+jute:
+	wget $(JUTE_URL)
+	go run github.com/go-zookeeper/jute/cmd/jutec \
+		-go.moduleMap=org.apache.zookeeper: \
+		-go.prefix=github.com/go-zookeeper/zk \
+		-outDir . \
```

Comment:
Maybe we should put these in an internal subdirectory?

Reply:
I am not sure about it. For example the lock recipe currently takes an `ACL` type.

```
func NewLock(c *Conn, path string, acl []ACL) *Lock {
```

This ACL type is generated by jute. So we want to be able to import this package in the recipes.